labs/week-01.md

```diff
@@ -0,0 +1,23 @@
+#Your Personal Portfolio Page: Part 1
+
+If you take a look at the "About Me" page you completed while applying for WDI, there are probably many things you would change. Today is an opportunity for you to change them! Strive for making this the first version of the Portfolio Page you use to exhibit your talent to prospective employers when WDI ends. Remember: you have the rest of the course (and the rest of your life) to revise it.
+
+## Schedule
+- 9:30 - 3:00 Lab time
+  - Supporting Instructors:
+    - 9:30 - 11:00 Erica, Jesse, Andy, Adrian
+    - 1:00 - 3:00 Erica, Robin, Matt, Nick
+- 3:00 - 4:00 Peer code review (Classrooms)
+- 4:00 - 5:00 Squad check-in (TBD)
+
+## Some Things To Consider
+- Think about who you want to view your site (potential employers, freelance opportunities, friends). I know it is tempting to make it flashy with tons of jQuery plugins, but the most important thing when designing your site is to make it EASY for your visitors to access the information they want and navigate around the page.
+- Think about the goal for your site and what you want your users to do (read about your story, look at your work/code/writing, contact you). Guide them to taking those actions by making them prominent / easy to find.
+- Potential employers will probably view your source code, so be sure to use proper indentation and organize your code.
+- Wireframe what you want your site to look like first!
```

Comment:
maybe a quick blurb here about what a wireframe is, or go over it in the lab delivery